internal/eval/commands.go

```diff
@@ -1037,6 +1037,15 @@ var (
 		IsMigrated: true,
 		NewEval:    evalZADD,
 	}
+	zcountCmdMeta = DiceCmdMeta{
+		Name: "ZCOUNT",
+		Info: `ZCOUNT key min max
+		Counts the number of members in a sorted set with scores between min and max (inclusive).
+		Use -inf and +inf for unbounded ranges. Returns 0 if the key does not exist.`,
+		Arity:      4,
```

Comment:
```suggestion
		Arity:      3,
```

Reply:
Why this is 3 @apoorvyadav1111 it's four right
1->Command itself
2->Key
3->Min
4->Max

or is it wrong?

Reply:
That is correct, no need to update!